### ISSUE TITLE
Add support for getting list of all selectable SKUs for a given App Service Plan

### ISSUE THREAD

**Issue by vabgujral** [Azure/azure-cli#13466]
**Is your feature request related to a problem? Please describe.**

Currently, there is no way to get a list of all the selectable SKUs for a given app service plan through Azure CLI. 

**Describe the solution you'd like**

It would be handy to have a CLI command to get the list of all the selectable SKUs for a given app service plan. Like **az appservice plan list-available-sku**. The commandlet will be similar to **az vm list-sku** which lists the Virtual machine SKUs available in a region. 

**Describe alternatives you've considered**

The only available option is to call Azure management REST API as explained here - https://docs.microsoft.com/en-us/rest/api/appservice/appserviceplans/getserverfarmskus

**Additional context**

I have been lately working on scaling my app services through Azure CLI command - **az appservice plan update**. It takes an input parameter called **sku**. Currently, there is no way in Azure CLI to get the list of available SKUs for the given app service plan that I can select from.

**Comment by yungezz** [comment]
routing to appropriate team

**Comment by ggailey777** [comment]
I also agree that SKUs should be listable both for App Service and Functions.

**Comment by SeaDude** [comment]
And other commonly deployed Azure Resources! Allow az cli calls with flag `--available-skus` to list all for a given resource (or the like).

**Comment by panchagnula** [comment]
We don't have an API for this so not something that can be handled reliably at this time.

**Comment by vabgujral** [comment]
@panchagnula Isn't this the API that needs to be used - https://docs.microsoft.com/en-us/rest/api/appservice/appserviceplans/getserverfarmskus

**Comment by panchagnula** [comment]
@vabgujral like I said *reliable* api - this currently has some known issues with Linux apps & windows container apps - so until those issues are fixed on API level we won't be taking the support on client side.

**Comment by gammelgaard52** [comment]
This is not solved yet, we still cannot get a list of those SKUs available, directly with a cmdlet from PowerShell or Azure CLI

**Comment by cheahcheesam** [comment]
Does Microsoft need any help on this? At this point I would be delighted if this feature gets delivered in 2 years time from now...

**Comment by seligj95** [comment]
> [!NOTE]
> **[AI-Assisted Triage]** This comment was generated with AI assistance as part of an App Service CLI issue review. Reply to this comment and tag @seligj95 to follow up.

**Summary**: Request to list selectable/available SKUs for an App Service Plan — users want to see which pricing tiers are available before creating or scaling a plan.

**Status**: Feature Approved

**Priority**: P3

**Next Steps**: Feature enhancement planned — adding a command or option to list available SKUs for App Service Plans.
